README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+We use the following icons for our organization and repository logos:
+- [Gear icon](https://thenounproject.com/search/?q=gear&i=102547) by Jardson Almeida, from the Noun Project
```

Comment:
So no extra line after the heading, but 1 after the paragraph?

Is this following any style guide or are we just making it the same as the gmusic.js readme style?